include/zephyr/bluetooth/gatt.h

```diff
@@ -981,6 +981,9 @@ struct bt_gatt_notify_params {
 	bt_gatt_complete_func_t func;
 	/** Notification Value callback user data */
 	void *user_data;
+#ifdef CONFIG_BT_EATT
+	enum bt_att_chan_option chan_option;
```

Comment:
I think we should add an entry in the release notes about the addition of this: Applications that enable `CONFIG_BT_EATT` will need to ensure that this is correctly set.